templates/repo/branch/list.tmpl

```diff
@@ -45,6 +46,19 @@
 										<p class="time">{{$.i18n.Tr "org.repo_updated"}} {{TimeSince .Commit.Committer.When $.i18n.Lang}}</p>
 									</td>
 									{{end}}
+									<td class="ui">
+										{{$totalCommits := addInteger .CommitsBehind .CommitsAhead}}
```

Comment:
Maybe one improvement to be done here.
It maybe better to not compute this in template if someone as a better idea.

I first glance I would suggest to change the sig of percentage to 
`func(n int, values ...int) float32` and make it compute n over / sum(values)
This will remove the need for an intermediate variable in template.